### PR TITLE
домашно?

### PR DESCRIPTION
**Pull request by DimoY**

### CODE REVIEW

**Comment by DimoY** [comment]
Домашно?

**Comment by DimoY** [comment]
Домашно?

File: homework/12_B/9_Dimo/homework1/CircleColision.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import math
```
Comment (svetlin-mladenov) [inline homework/12_B/9_Dimo/homework1/CircleColision.py:1]: You sure your number in class is 9? Isn't it 7? Are you from Б class?

File: homework/12_B/9_Dimo/homework1/CircleColision.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+class ColisionType:
```
Comment (svetlin-mladenov) [inline homework/12_B/9_Dimo/homework1/CircleColision.py:3]: You can replace all of these classes with a single enum.

File: homework/12_B/9_Dimo/homework1/CircleColision.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+class Circle:
+    def __init__(self,x,y,r) -> None:
+        self.x = x
+        self.y = y 
+        self.r = r
+    def __str__(self) -> str:
+        return "Circle at x:"+str(self.x)+" and y:"+str(self.y)+" and radius:"+str(self.r)
+    def circleColide(self,c2) ->ColisionType:
+        d = math.sqrt((self.x-c2.x)**2 + (self.y-c2.y)**2)
+        if d <= self.r-c2.r:
```
Comment (svetlin-mladenov) [inline homework/12_B/9_Dimo/homework1/CircleColision.py:46]: What if `d == self.r - c2.r`

**Review by svetlin-mladenov — CHANGES_REQUESTED** [review]

File: homework/12_B/9_Dimo/homework1/CircleColision.py (hunk 3 above)
Comment (DimoY) [reply homework/12_B/9_Dimo/homework1/CircleColision.py]: What about line 46?

Comment (DimoY) [reply homework/12_B/9_Dimo/homework1/CircleColision.py]: :+1:  :shipit:

File: homework/12_B/9_Dimo/homework1/CircleColision.py (hunk 1 above)
Comment (svetlin-mladenov) [reply homework/12_B/9_Dimo/homework1/CircleColision.py]: OK. I see you've amended that.

File: homework/12_B/9_Dimo/homework1/CircleColision.py (hunk 3 above)
Comment (svetlin-mladenov) [reply homework/12_B/9_Dimo/homework1/CircleColision.py]: When is `d == self.r - c2.r`? It is a special condition.

**Pull request merged**